### PR TITLE
remove infinity hover

### PR DESCRIPTION
**Pull request by Ritika8081**
@akadeepesh Hover is removed, Now you can click on infinity button then you will be able to see description of time.

### CODE REVIEW

File: src/components/Connection.tsx [hunk 1]
```diff
@@ -369,63 +374,63 @@ const Connection: React.FC<ConnectionProps> = ({
...
+          size="sm"
+          onClick={() => handleTimeSelection(null)}
+        >
+          <Infinity className="h-4 w-4" />
+        </Button>
+      </div>
+    </div>
+  </PopoverContent>
+</Popover>
             </div>
```
Comment (akadeepesh) [inline src/components/Connection.tsx]: @Ritika8081 Please use prettier as code formatter to maintain the indentation of code. the spacing should not be changed. Also please add a screen shot of button in open state.

**Comment by Ritika8081** [comment]
@akadeepesh fixed indentation issues.

File: src/components/Connection.tsx (hunk 1 above)
Comment (Ritika8081) [reply src/components/Connection.tsx]: fixed issue.

**Review by akadeepesh — COMMENTED** [review]
Again that code format thing, please use prettier or make the whole file in same code format. Spacing and indentation should be followed commonly.

File: src/components/Connection.tsx [hunk 2]
```diff
@@ -513,21 +515,57 @@ const Connection: React.FC<ConnectionProps> = ({
...
-                    <span className="flex flex-row justify-center items-center">
-                      <FileArchive />
-                      <p className=" text-lg">{`(${datasets.length})`}</p>
-                    </span>
-                  )}
-                </TooltipTrigger>
-              </Button>
+              <div className="flex">
+                <Button onClick={saveData} className="rounded-r-none">
+                  <TooltipTrigger asChild>
```
Comment (akadeepesh) [inline src/components/Connection.tsx:525]: Can you add a image how the button UI looks after rounded-r-none

File: src/components/Connection.tsx [hunk 3]
```diff
@@ -38,6 +38,7 @@ import {
   PopoverContent,
   PopoverTrigger,
 } from "@/components/ui/popover";
+import { ArrowUp, Trash2, Download } from "lucide-react";
```
Comment (akadeepesh) [inline src/components/Connection.tsx]: We have already imported some icons from lucide-react, you should add these three icons in that same import.

**Review by akadeepesh — CHANGES_REQUESTED** [review]

**Review by akadeepesh — APPROVED** [review]

**Comment by akadeepesh** [comment]
@Ritika8081 please confirm that this PR is ready to merge, if not. Make a ready to merge commit, you can revert doubtful changes

**Comment by Ritika8081** [comment]
you can merge now.

**Pull request merged**